pkg/apis/channels/v1alpha1/bus_defaults.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+func (b *Bus) SetDefaults() {
```

Comment:
It's fine for these no-op methods to exist but I'd like to see a comment justifying their existence, otherwise someone will remove them unknowingly. Do we expect defaults to be added in the near future? Is this a demonstration of what's possible? A desire for consistency across all resource types? A need for all objects to implement the same interface?

Reply:
[n3wscott]: Note about #321 and other related Validatable stub PRs: This is staging work, the plan is another pass to bring up the test coverage, then remove unused after each type is stubbed.

Reply:
Makes sense. Can you add those words as a comment in this file? Just in case this work ends up taking longer than expected. 😸